config/module.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/spf13/cobra"
+)
+
+// Module defines an interface of Module
```

Comment:
Not `// Module defines...`
but `// ModuleI defines...`

Reply:
Fixed at https://github.com/hyperledger-labs/yui-relayer/pull/36/commits/2341396e26e41664ecc620e80b47284757dc3e27